packages/tools/devtools/devtools-browser-extension/src/popup/PopupScript.ts

```diff
@@ -28,4 +51,24 @@ browser.tabs.query({ active: true, currentWindow: true }, (tab) => {
 		'To use the Fluid Devtools, open the browser Devtools pane (F12) and click the "Fluid Developer Tools" tab.';
 
 	document.body.append(popupElement);
+	initializePopupView(popupElement, tabId).then(() => {
+		console.debug(`Rendered popup for tab ${tabId}!`);
+	}, console.error);
 });
+
+/**
+ * Renders the Fluid Popup view into the provided target element.
+ *
+ * @param target - The element into which the popup view will be rendered.
+ */
+export async function initializePopupView(target: HTMLElement, tabId: number): Promise<void> {
+	const backgroundServiceConnection = await BackgroundConnection.Initialize({
+		// TODO: devtools-panel-specific source
```

Comment:
I would recommend we make this change before merging - it will help us determine whether requests are coming from the popup or the devtools view when debugging.

Reply:
We just need to be careful to ensure existing flows continue to work (manual testing + auditing code to check for any places checking for the specific existing source here).